ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4760,4 +4760,20 @@ HWRWISOVPM:
   - heating_return_water_isolation_valve_percentage_command
   - heating_return_water_isolation_valve_percentage_sensor
   implements:
-  - MONITORING
\ No newline at end of file
+  - MONITORING 
+
+DSS:
```

Comment:
use something like `DEFSS` and include the run command as well.

Reply:
DEFSS:
  description: "defrost run command and status (start/stop) "
  is_abstract: true
  uses:
  - defrost_run_status
  - defrost_run_command  
  implements:
  - MONITORING